eng/pipelines/live-test-cleanup.yml

```diff
@@ -52,3 +52,13 @@ stages:
           @subscriptionConfiguration `
           -Verbose
         displayName: Clean up subscription resources (AzureChinaCloud)
+
+      - pwsh: |
+          $subscriptionConfiguration = @'
+            $(acs-subscription-configuration)
+          '@ | ConvertFrom-Json -AsHashtable
```

Comment:
We're almost to the point where it might make sense to templatize this and iterate through parameters, but this is great for now.